internal/testutil/utils.go

```diff
@@ -346,89 +346,84 @@ func (te *TestEtcd) WaitDown(timeout time.Duration) error {
 
 const (
 	giteaAppIniTmpl = `
-APP_NAME = Gitea: Git with a cup of tea
-RUN_MODE = prod
-RUN_USER = {{ .User }}
-
-[repository]
-ROOT = {{ .Data }}/git/repositories
-
-[repository.local]
-LOCAL_COPY_PATH = {{ .Data }}/gitea/tmp/local-repo
-
-[repository.upload]
-TEMP_PATH = {{ .Data }}/gitea/uploads
-
-[server]
-APP_DATA_PATH    = {{ .Data }}/gitea
-SSH_DOMAIN       = {{ .SSHListenAddress }}
-HTTP_PORT        = {{ .HTTPPort }}
-ROOT_URL         = http://{{ .HTTPListenAddress }}:{{ .HTTPPort }}/
-DISABLE_SSH      = false
-# Use built-in ssh server
-START_SSH_SERVER = true
-SSH_PORT         = {{ .SSHPort }}
-LFS_CONTENT_PATH = {{ .Data }}/git/lfs
-DOMAIN           = localhost
-LFS_START_SERVER = true
-LFS_JWT_SECRET   = PI0Tfn0OcYpzpNb_u11JdoUfDbsMa2x6paWH2ckMVrw
-OFFLINE_MODE     = false
-
-[database]
-PATH     = {{ .Data }}/gitea/gitea.db
-DB_TYPE  = sqlite3
-HOST     =
-NAME     =
-USER     =
-PASSWD   =
-SSL_MODE = disable
-
-[indexer]
-ISSUE_INDEXER_PATH = {{ .Data }}/gitea/indexers/issues.bleve
-
-[session]
-PROVIDER_CONFIG = {{ .Data }}/gitea/sessions
-PROVIDER        = file
-
-[picture]
-AVATAR_UPLOAD_PATH      = {{ .Data }}/gitea/avatars
-DISABLE_GRAVATAR        = false
-ENABLE_FEDERATED_AVATAR = true
-
-[attachment]
-PATH = {{ .Data }}/gitea/attachments
-
-[log]
-ROOT_PATH = {{ .Data }}/gitea/log
-MODE      = file
-LEVEL     = info
-
-[security]
-INSTALL_LOCK   = true
-SECRET_KEY     = vRCH8usxWj6e8JGBPBaqycpfVyWm079xC3P3k76YsjKbrgBmyHhQD9UyzRFICKBT
-INTERNAL_TOKEN = eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYmYiOjE1NTc0MDI0MDZ9.27f4bakIxBIOoO48ORyLmbvpQprsJMEHLM6PyXIqB5g
-
-[service]
-DISABLE_REGISTRATION              = false
-REQUIRE_SIGNIN_VIEW               = false
-REGISTER_EMAIL_CONFIRM            = false
-ENABLE_NOTIFY_MAIL                = false
-ALLOW_ONLY_EXTERNAL_REGISTRATION  = false
-ENABLE_CAPTCHA                    = false
-DEFAULT_KEEP_EMAIL_PRIVATE        = false
-DEFAULT_ALLOW_CREATE_ORGANIZATION = true
-DEFAULT_ENABLE_TIMETRACKING       = true
-NO_REPLY_ADDRESS                  = noreply.example.org
-
-[oauth2]
-JWT_SECRET = hQdtj6H6lsd8vG6V1vCPYcOn8uP2C3i_bbnDozfCcIY
-
-[mailer]
-ENABLED = false
-
-[openid]
-ENABLE_OPENID_SIGNIN = true
-ENABLE_OPENID_SIGNUP = true
+	APP_NAME = Gitea: Git with a cup of tea
```

Comment:
Don't add tabs at the start since the generated file will have these spaces too.